tasks/main.yml

```diff
@@ -1 +1,36 @@
 # role tasks
+
+- name: Find the current locale setting
+  slurp:
+    src: /etc/locale.conf
```

Comment:
How large are this and the other flies you're slurping likely to be? When you use `slurp` to fetch the entire file, we want to be careful that the files aren't going to grow to a large size and take up a bunch of bandwidth if our need is only for a single line or two in the file.

Reply:
The two remaining files that I need to use slurp on are quite small